openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -829,20 +830,29 @@ class TestAccountDeactivation(TestCase):
 
     def setUp(self):
         super(TestAccountDeactivation, self).setUp()
-        self.superuser = SuperuserFactory()
-        self.staff_user = AdminFactory()
         self.test_user = UserFactory()
         self.url = reverse('accounts_deactivation', kwargs={'username': self.test_user.username})
 
-    def assert_activation_status(self, expected_status=status.HTTP_200_OK, expected_activation_status=False):
+    def build_jwt_headers(self, user):
+        """
+        Helper function for creating headers for the JWT authentication.
+        """
+        expires_in = settings.OAUTH_ID_TOKEN_EXPIRATION
```

Comment:
Remove this, and the customization of `expires_in`. The `build_token()` uses the JWT expiration time by default. Also, while ID tokens (used for OpenID Connect) are JWTs, they are meant to expire much more quickly (30s) than normal tokens (1h).